reference-architectures/extension/deploy-arch-ibm-pvs-inf-extension.md

```diff
@@ -0,0 +1,133 @@
+---
+copyright:
+  years: 2023
+lastupdated: "2023-02-21"
+
+keywords:
+
+subcollection: deployable-reference-architectures
+
+authors:
+  - name: Arnold Beilmann
+
+version: v7.0.0
+
+deployment-url: url #TODO
+
+docs: https://cloud.ibm.com/docs/solution-guide #TODO
+
+image_source: https://github.com/terraform-ibm-modules/terraform-ibm-powervs-infrastructure/reference-architectures/extension/deploy-arch-ibm-pvs-inf-extension.svg
+
+related_links:
+  - title: 'SAP in IBM Cloud documentation'
+    url: 'https://cloud.ibm.com/docs/sap'
+    description: 'SAP in IBM Cloud documentation.'
+  - title: 'Reference architecture for "Secure infrastructure on VPC for regulated industries"'
+    url: 'https://url' #TODO
+    description: 'Reference architecture for "Secure infrastructure on VPC for regulated industries"'
+
+use-case: ITServiceManagement
+
+industry: Technology
+
+compliance: SAPCertified
+
+content-type: reference-architecture
+
+---
+
+{{site.data.keyword.attribute-definition-list}}
+
+# Power infrastructure for deployable architectures - variation 'PowerVS workspace'
+{: #deploy-arch-ibm-pvs-inf-standard}
+{: toc-content-type="reference-architecture"}
+{: toc-industry="Technology"}
+{: toc-use-case="ITServiceManagement"}
+{: toc-compliance="SAPCertified"}
+
+Building upon the VPC service that were previously created when you deployed 'Secure infrastructure on VPC for regulated industries' or 'Power Infrastructure for deployable architecture' in variation 'full-stack', 'Power Infrastructure for deployable architectures' in variation 'extension' creates a Power Virtual Server workspace and connects it with VPC services. Proxy service for public internet access from PowerVS workspace is configured.
+
+You can optionally (re)configure management components on VPC (such as NFS server, NTP forwarder, and DNS forwarder).
```

Comment:
You can optionally reconfigure management components on VPC (such as NFS server, NTP forwarder, and DNS forwarder).

Reply:
Had a similar suggestion